backend/inspirehep/records/tasks.py

```diff
@@ -43,3 +54,64 @@ def update_records_relations(uuids):
 
     db.session.commit()
     return uuids
+
+
+@shared_task(
+    bind=True,
+    acks_late=True,
+    retry_backoff=True,
+    retry_kwargs={"max_retries": 3},
+    autoretry_for=(OperationalError, RequestError, TransportError),
```

Comment:
* I would put this in the `indexer_task` queue, as it's the same kind of stuff.
* `RequestError` is a subclass of `TransportError` if I'm not mistaken.